server/viame_server/serializers/viame.py

```diff
@@ -148,9 +148,12 @@ def _parse_row_for_tracks(row: List[str]) -> Tuple[Feature, Dict, Dict, List]:
         fishLength=fishLength if fishLength > 0 else None,
         **head_tail_feature,
     )
+    sorted_confidence_pairs = sorted(
+        confidence_pairs, key=lambda item: item[1], reverse=True
+    )
```

Comment:
What do you think about moving this sort into `_parse_row` right after `confidence_pairs` is created?

Wouldn't want inconsistent sort behavior to crop up, for example if some other function called `_parse_row`.

Reply:
Yeah, that was an oversight on my part.  I'll move it into the `_parse_row` and do some testing again.